docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+this field should contain a URI to the corresponding record in the register. There can be multiple registers.
+A URI could look like `https://huggingface.co/org/modelid` or `https://algoritmes.overheid.nl/nl/algoritme/modelid`.
+2. `language` (OPTIONAL, list[string]). If relevant, the natural languages the model supports in ISO 639.
+    There can be multiple languages.
+3. `license`(REQUIRED, string). Any license from the [license list](https://huggingface.co/docs/hub/repositories-licenses).
+If the license is NOT present in the license list this field must be set to 'other' and the following two fields will
+be REQUIRED.
+
+    1. `license_name` (string). An id for the license.
+    2. `license_link` (string). A link to a file of that name inside the repo, or a URL to a remote file containing the license
+    contents.
+
+4. `library` (OPTIONAL, string). Any library from the [library list](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts).
+5. `tags` (OPTIONAL, list[string]). Tags with keywords to describe the project. There can be multiple tags.
+6. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+    it should contain a URI from the [Uniform Product List](https://standaarden.overheid.nl/owms/oquery/UPL-actueel.plain).
+7. `owners` (list). There can be multiple owners. For each owner the following fields are present.
+
+    1. `organization` (REQUIRED, string). Name of the organization that owns the model.
+    2. `oin` (OPTIONAL, string). If applicable the [Organisatie-identificatienummer (OIN)](https://www.logius.nl/domeinen/toegang/organisatie-identificatienummer/wat-is-het).
+    3. `name` (OPTIONAL, string). Name of a contact person within the organisation.
+    4. `email` (OPTIONAL, string). Email address of the contact person or organization.
+    5. `role` (OPTIONAL, string). Role of the contact person. This field should only be set when the `name` field
+    is set.
+
+### 2. `model_index`
+
+There can be multiple models. For each model the following fields are present.
+
+1. `name` (REQUIRED, string). The name of the model.
+2. `model` (REQUIRED, string). A URI pointing to a repository containing the model file.
+3. `results` (list). There can be multiple results. For each result the following fields are present.
+
+    1. `task` (list).
+
+        1. `task_type` (REQUIRED, string). The task of the model, for example "object-classifcation".
+        2. `task_name` (OPTIONAL, string). A pretty name fo the model taks, for example "Object Classification".
+
+    2. `dataset` (list).
+
+        1. `type` (REQUIRED, string). The type of the dataset, can be a dataset id from [HuggingFace datasets](https://hf.co/datasets)
+        or a link to a repository containing the dataset[^1], for example "common_voice".
+        2. `name` (REQUIRED, string). Name pretty name for the dataset, for example "Common Voice (French)".
```

Comment:
Just like with the metrics i could also see that we maybe want to show some features of the dataset, but this is more for a dataset modelcard to be honest

Reply:
We could include it. Anne also did it in in the modelcard in his POC.